README.md

```diff
@@ -10,7 +10,8 @@ The New Relic Lambda Tracer is an OpenTracing Tracer implementation specifically
 
 ## Installation
 
-See our [agent installation documentation](https://docs.newrelic.com/docs/agents/net-agent/installation/introduction-net-agent-install).
+<!-- markdown-link-check-disable-next-line -->
+The latest released binaries for the .NET Agent can be found either [here](http://download.newrelic.com/dot_net_agent/latest_release/) or on [NuGet.org](https://www.nuget.org/profiles/newrelic).  See our [agent installation documentation](https://docs.newrelic.com/docs/agents/net-agent/installation/introduction-net-agent-install) to understand which of these files are appropriate for your application type and hosting environment.
```

Comment:
Could you make sure that the sentence "See our ..." on the next line.?  I cannot tell from the diff comparison if it is continuing directly from the URL of Nuget.org.

Reply:
Good idea, done.